frontend/src/features/projects/erp/components/ExemptionDetails.tsx

```diff
@@ -0,0 +1,38 @@
+import { IProject, ProjectNotes } from 'features/projects/common';
+import { Form, FastDatePicker } from 'components/common/form';
+import React from 'react';
+import { useFormikContext } from 'formik';
+
+interface IExemptionDetailsProps {
+  isReadOnly?: boolean;
+}
+
+/**
+ * Provides project exemption form fields.
+ * @param param0 Component properties.
```

Comment:
Not that important but should `param0` -> `isReadOnly`?

Reply:
One would think, but the argument is actually an anonymous object that extracts `isReadonly`.  I should however use jsdoc to clearly identify `param0.isReadonly`.